test/integration/json.jl

```diff
@@ -19,9 +19,19 @@
         @test json_response isa JSON3.Object
         @test json_response == JSON3.read(raw_response)
 
-        dict_response = FHIRClient.request(Dict, client, "GET", patient_request)
+        test_logger = TestLogger(; min_level = Logging.Debug)
+        dict_response = Logging.with_logger(test_logger) do
+            FHIRClient.request(Dict, client, "GET", patient_request)
+        end
         @test dict_response isa Dict
         @test dict_response == JSON3.read(raw_response, Dict)
+        log = only(test_logger.logs)
+        @test log.message == "FHIRClient.request()"
+        @test log.level == Logging.Debug
```

Comment:
```suggestion
        @test log.level == LogLevel(-1_000)
```

Reply:
I applied this change in https://github.com/JuliaHealth/FHIRClient.jl/pull/156/commits/b7adb730abcddacbbea290a8e772853cf7fe995e but did not commit your suggestion on Github since `LogLevel` has to be qualified 🙂